### DISCUSSION TITLE
Remote React Query Dev Tools release - React Native supported!

### DISCUSSION THREAD

**Discussion by LovesWorking** [TanStack/query#6890] [Show and tell]
![react-query-external-preview](https://github.com/LovesWorking/LovesWorking/assets/111514077/e8c119cc-44bc-48ba-a398-dfba30e44396)
https://github.com/LovesWorking/react-query-external-sync

I've developed React Query External Sync, a tool enabling React Query DevTools use with React Native via an external dashboard.

I'm wondering if it could be featured on the [React Query DevTools documentation page] (https://tanstack.com/query/v4/docs/framework/react/devtools). I believe it could provide significant value to the React Native community.

Thank you!

@lachlancollins

**Comment by TkDodo** [comment]
sure, please make a PR to the docs.

**Comment by frankcalise** [comment]
Awesome work @LovesWorking! I'm wondering if we could pull your dash into [Reactotron](https://github.com/infinitered/reactotron/)

**Reply by LovesWorking** [reply]
> Awesome work @LovesWorking! I'm wondering if we could pull your dash into [Reactotron](https://github.com/infinitered/reactotron/)

Definitely! Looks awesome. Let me know how it goes or if you need any help.
